Firestore/Example/Tests/Local/FSTLocalStoreTests.mm

```diff
@@ -41,16 +43,23 @@
 #import "Firestore/third_party/Immutable/Tests/FSTImmutableSortedSet+Testing.h"
 
 #include "Firestore/core/src/firebase/firestore/auth/user.h"
+#include "Firestore/core/src/firebase/firestore/model/document_dictionary.h"
+#include "Firestore/core/src/firebase/firestore/model/document_key_set.h"
+#include "Firestore/core/src/firebase/firestore/model/document_version_dictionary.h"
 
 using firebase::firestore::auth::User;
+using firebase::firestore::model::DocumentDictionary;
+using firebase::firestore::model::DocumentKeySet;
+using firebase::firestore::model::DocumentVersionDictionary;
+using firebase::firestore::model::MaybeDocumentDictionary;
 
 NS_ASSUME_NONNULL_BEGIN
 
 /** Creates a document version dictionary mapping the document in @a mutation to @a version. */
-FSTDocumentVersionDictionary *FSTVersionDictionary(FSTMutation *mutation,
-                                                   FSTTestSnapshotVersion version) {
-  FSTDocumentVersionDictionary *result = [FSTDocumentVersionDictionary documentVersionDictionary];
-  result = [result dictionaryBySettingObject:FSTTestVersion(version) forKey:mutation.key];
+DocumentVersionDictionary FSTVersionDictionary(FSTMutation *mutation,
```

Comment:
Note that you've replaced an immutable map (FSTDocumentVersionDictionary) with a mutable one (DocumentVersionDictionary.) Check out Firestore/core/src/f/f/immutable. You might find something in there more suitable than a raw mutable map for this purpose.